lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>jako technicky vzato a správně a pokud navíc má člověk kód napsán výše, tak je to i rychlé pro úpravu, ale striktně vzato, asi bych spíš filtroval a pak aplikoval podmínku</p><p> </p><p>nevím, jestli to stojí za zmínku, jen jsem měl potřebu si rejpnout :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
<p><code>countries.loc["Czechia", "alcohol_adults"] &gt; 15</code>  </p><p> </p><p>je určitě lepší. Díky</p>